site/src/my-entries.md

```diff
@@ -13,8 +13,16 @@ Within your Sample's box, you can see the information you recorded, as well as a
 - Create a duplicate of the entry
 - And add new, different kinds of entries that reference this entry, such as Serial Dilution Plates and Isolates. More information about the various types of database entries are described in the "Entry Types" section of this workshop
 
+By clicking the box that says "add media", you can add photos to support your written data. 
+
+Photos can add details about your soil sample and where it was collected. 
+
+You will also be able to set a featured image, which means the image that will first appear at the top of all pages for this sample. 
+
+You can find the "Set featured image" button to the top right corner of the form. 
+
```

Comment:
I think you meant to add this note to `site/src/first-entry.md`

This page, `site/src/my-entries.md`, is for the My Entries page which shows an overview of all your data, not any of the form fields for directly editing or creating entries